vision/common/constants.py

```diff
@@ -23,6 +23,25 @@
 Corners: TypeAlias = NDArray[Shape["4, 2"], Float64]
 
 
+class Location(TypedDict):
+    """
+    A saved coordinate location - primarily used for ODLCs
+
+    Attributes
+    ----------
+    latitude: float
+        The latitude in degrees of the location
+    longitude: float
+        The longitude in degrees of the location
+    """
+
+    latitude: float
+    longitude: float
+
+
+ODLC_Dict: TypeAlias = dict[int, Location]
```

Comment:
Not really sure what this is for. Could you provide clarification with a comment? Also, if this is to store info about standard objects, should consider using the `StandardObject` described above instead of `Location` for the item value type.